src/marker_on_image.cpp

```diff
@@ -18,6 +18,16 @@ struct MarkerInfo
   ros::Time expiration;
 };
 
+/// Holds information about a corner of a cube marker.
+struct CubeCorner
```

Comment:
Since the data structure is more generic than just be able to store cube corners how about renaming it to something like "PoseProjection"?

Reply:
Good idea, done in https://github.com/osrf/marker_on_image/pull/1/commits/ae23e01d3253f17ef130b1c3f4d4fd8d85cf60ee